metals/src/main/scala/scala/meta/internal/troubleshoot/ScalaProblem.scala

```diff
@@ -0,0 +1,75 @@
+package scala.meta.internal.troubleshoot
+
+import scala.meta.internal.metals.BuildInfo
+import scala.meta.internal.metals.ScalaVersions
+
+sealed abstract class ScalaProblem {
+  def message: String
+
+  protected val hint = "run 'Build import' to enable code navigation."
+
+}
+
+// scala version issues
```

Comment:
I think we can probably just remove this comment or include proper scaladocs for these.

Also related, if possible can you add a short Scaladoc to `ScalaProblem`. I find that when all of the classes have short descriptions it makes it much easier for someone new to the codebase to get a quick idea of what a class is meant for then.

Reply:
I removed the smaller comments and add scaladoc to `ScalaProblem`